### PR TITLE
Postman #75

### LINKED ISSUE #75: group Postman? Insomia?

### PR DESCRIPTION
**Pull request by Ryszard-S**
close #75

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,3 +51,7 @@ To restart the data to init state use:
 If you have some problems with removing old containers use:
 
 `yes | docker-compose rm`
+
+### POSTMAN
+
+import file camp-trello-clone.postman_collection to postman
```
Comment (kacper-cyra) [inline README.md:57]: I would love to see how to do it, or get some link to docs how to work with it (import, export) to narrow the time to do it

File: camp-trello-clone.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (mcwiekala) [inline camp-trello-clone.postman_collection.json:1]: move to `/postman/camp-trello-clone.postman_collection.json`

File: README.md (hunk 1 above)
Comment (mcwiekala) [inline README.md:57]: create properties file with port, and commit to repository - PORT = 8085

Comment (mcwiekala) [inline README.md:57]: create properties file with port, and commit to repository - PORT = 8085

Comment (mcwiekala) [inline README.md:57]: create properties file with port, and commit to repository - PORT = 8085

**Review by mcwiekala — APPROVED** [review]

**Pull request merged**